core/src/main/java/org/apache/brooklyn/core/internal/BrooklynProperties.java

```diff
@@ -181,6 +181,7 @@ public static Builder fromProperties(BrooklynProperties brooklynProperties) {
             }
 
             @Override
+            @SuppressWarnings("deprecation")
```

Comment:
Minor - would be better to change `toString` to use `MoreObjects` rather than the deprecated `Objects` rather than silencing the warning.